src/FullBook/FullBook.css

```diff
@@ -0,0 +1,22 @@
+.full-book-container {
+  height: 55rem;
+  max-width: 55rem;
+  /* display: inline-block; */
+  /* vertical-align: middle; */
+  position: relative;
+  left: 50%;
+  transform: translateX(-50%);
+  margin-top: 14rem;
+  margin-bottom: 11rem;
+  /* height: 75rem; */
+}
+
+embed {
```

Comment:
Whoa, what is `embed`?

Reply:
That's for rendering actual files and other odd things in your window.  You can also use iframe.